cloud-native-app-developer/reading-list-front-end-with-managed-auth/src/index.tsx

```diff
@@ -155,6 +155,9 @@ export default function App() {
       <div className="py-3 md:py-6">
         <div className="container px-4 mx-auto flex justify-center">
           <div className="w-full max-w-lg px-2 py-16 sm:px-0 mb-20">
+            <div className='flex'>
+              <p className="text-4xl text-white mb-3 font-bold">Welcome {user?.email}</p>
```

Comment:
shall we add `id` for this element and check by it.

Reply:
Fixed